.github/workflows/build.yml

```diff
@@ -277,7 +277,7 @@ jobs:
     steps:
     - name: Delete image artifacts
       if: ${{ !env.INTERNAL_CONTRIBUTOR }}
-      uses: geekyeggo/delete-artifact@v1
+      uses: geekyeggo/delete-artifact@v4
```

Comment:
If this version is out of sync with the "upload-artifact" version then the command fails quietly.